src/easystar.js

```diff
@@ -286,6 +290,23 @@ EasyStar.js = function() {
             instance.startY, null, STRAIGHT_COST));
 
         instances.push(instance);
+        return instance.id;
+    };
+
+    /**
+     * Cancel a path calculation.
+     *
+     * @param {Number} id The ID of the path being calculated.
+     *
+     **/
+    this.cancelPath = function(id) {
+        for (var i = 0; i < instances.length; i ++) {
+            var instance = instances[i];
```

Comment:
Actually would you mind creating a map for this? As you said, it is probably a negligible performance benefit in practice, but objects are cheap so why not?